erts/emulator/test/jit_SUITE.erl

```diff
@@ -18,16 +18,17 @@
 %% %CopyrightEnd%
 %%
 
--module(perf_SUITE).
+-module(jit_SUITE).
 
--export([all/0, suite/0, init_per_suite/1, end_per_suite/1]).
--export([symbols/1, annotate/1]).
+-export([all/0, suite/0, init_per_suite/1, init_per_testcase/2,
+	 end_per_suite/1]).
+-export([symbols/1, annotate/1, named_labels/1]).
 
 suite() ->
     [{timetrap, {minutes, 4}}].
 
 all() -> 
-    [symbols, annotate].
+    [symbols, annotate, named_labels].
```

Comment:
Put the perf related testcases in a perf group and then change the init_per_suite to be a init_per_group so that the correct testcases are skipped on the correct platforms.

```suggestion
    [{group, perf}, named_labels].
    
groups() ->
  [{perf,[symbols, annotate]}].
```

Reply:
Done.